src/components/ui/copy/copy.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled, { css } from 'styled-components';
+import { IconButton } from '../buttons/icon-button';
+import { Icon } from '../icon/icon';
+
+const Default_Clicked_Duration = 3000;
+
+interface CopyProps {
+  /**
+   * Duration, that the Checkmark will be shown once the button is clicked
+   * @default 3000ms
+   */
+  clickedDuration?: number;
```

Comment:
This name doesn't resonate with me, what about `feedbackMessageTime`? The `click` is of course the action someone did before but describing the thing, the "feedback message" makes it more readable. You can also omit `copy` because that's the context of the entire component anyway.

Reply:
@feedm3 suggested showSuccessNotificationMS, I have now used that

Reply:
thanks! Mind the `MS` postfix noting the unit format of milliseconds. That's a good point too 👍 I agree with the name.